freesound/settings.py

```diff
@@ -171,6 +171,17 @@
 MAX_UNMODERATED_SOUNDS_IN_HOME_PAGE = 5
 ALLOWED_AUDIOFILE_EXTENSIONS = ['wav', 'aiff', 'aif', 'ogg', 'flac', 'mp3']
 
+# After download modal
+AFTER_DOWNLOAD_MODAL = 'survey'  # Set it to none for no modal after download
```

Comment:
Sure, I kind of forget abut these often... fixed in https://github.com/MTG/freesound/pull/820/commits/a9e4df7e7f9acd995393a36e52ede59768a4de05